spec/system/planning_applications/sitemap_spec.rb

```diff
@@ -156,7 +156,7 @@
       expect(map["showMarker"]).to eq("true")
 
       find(".govuk-visually-hidden",
-        visible: false).set '{"type":"Feature","properties":{},"geometry":{"type":"Polygon","coordinates":[[[-0.076715,51.501166],[-0.07695,51.500673],[-0.076,51.500763],[-0.076715,51.501166]]]}}'
+        visible: false).set({"type" => "Feature", "properties" => {}, "geometry" => {"type" => "Polygon", "coordinates" => [[[-0.076715, 51.501166], [-0.07695, 51.500673], [-0.076, 51.500763], [-0.076715, 51.501166]]]}})
```

Comment:
This is setting it to a hash string, not a json string so is probably why you need the `gsub` calls in the setter method